source/manual/post-a-statuspage-message.html.md

```diff
@@ -4,27 +4,32 @@ title: Post a message on the status page
 section: Incidents
 layout: manual_layout
 parent: "/manual.html"
-last_reviewed_on: 2018-08-01
+last_reviewed_on: 2018-11-02
 review_in: 3 months
 important: true
 ---
 
-When something goes wrong on GOV.UK we follow our [incident management procedure][inc]. As part of that we update our status page [status.publishing.service.gov.uk][status]
+When something goes wrong on GOV.UK we follow our [incident management procedure][inc].
+As part of that we update our status page [status.publishing.service.gov.uk][status].
 
 The page is managed using a third party service called [Statuspage][sp].
 
 Use [manage.statuspage.io][man] to post messages.
 
-Remember: government publishers are the main audience of the page. They'll want to know what isn't working, and how long it will take to fix. Write in non-technical language. Use the pre-written templates, they've been proofread and can be customised to most situations.
+Remember: government publishers are the main audience of the page.
+They'll want to know what isn't working, and how long it will take to fix. Write in non-technical language.
+Use the pre-written templates, they've been proofread and can be customised to most situations.
 
 This is what the [Statuspage admin interface][man] looks like:
 
-![](manual/images/create-new-incident.png)
+![](images/create-new-incident.png)
 
-## Title of the incident
+## Incident Name
```

Comment:
I don' think we use Title Case in headings unless the words within themselves are titles, so `Incident name` would be more appropriate.

Reply:
I see how this looks like Title Case. That wasn't my intention, though. I only followed the names of the sections in https://manage.statuspage.io/pages/xcz7hfscrbjs/incidents/new. They are:
- `Incident Name`
- `Incident Status`
- `Message`

Reply:
Ah ok. I think America defaults to Title Case on everything, but I'm not going to hold this up on that front. Thanks for clarifying!